Nodejs/Product/Nodejs/Resources.resx

```diff
@@ -600,4 +600,13 @@ You will need to restart Visual Studio after installation.</value>
   <data name="SurroundWith" xml:space="preserve">
     <value>Surround With</value>
   </data>
+  <data name="TsdErrorOccured" xml:space="preserve">
+    <value>Error installing typings.</value>
+  </data>
+  <data name="TsdInstallCompleted" xml:space="preserve">
+    <value>Successfully installed typings.</value>
+  </data>
+  <data name="TsdNotInstalledError" xml:space="preserve">
+    <value>Could not find TSD. Make sure you have installed it globally.</value>
```

Comment:
It's best to provide people with resolution steps whenever there are errors rather than just telling them what the error is, and pinpoint exactly what the error is. Additionally, in user-land, tsd means nothing to people. What is tsd? Why do I need it? So let's try to simplify this to user-language `"Error installing IntelliSense for package {0}: {1}", packageName, errorDescriptionAndResolution.` Same thing with the rest of these.